src/Forms/Main.cs

```diff
@@ -19081,19 +19097,50 @@ private void exportCustomTextFormatToolStripMenuItem_Click(object sender, EventA
 
         private void PasteIntoActiveTextBox(string s)
         {
-            if (tabControlSubtitle.SelectedIndex == TabControlSourceView)
+            if (SubtitleListview1.SelectedItems.Count > 1)

```

Comment:
Hm, also if you insert something in a text box (by right click in the text box) it could be inserted into multiple lines which will be rather unexpected I think